apps/music/open.html

```diff
@@ -22,6 +22,7 @@
   <script defer src="/shared/js/intl/l20n.js" type="application/javascript;version=1.8"></script>
   <script defer src="/shared/js/intl_helper.js"></script>
   <script defer src="/components/gaia-header/dist/gaia-header.js"></script>
+  <script defer src="/elements/music-scan-progress.js"></script>
```

Comment:
Why do we need the scan progress? My expectation would be that the open activity never even *looks* at the database.